ios/CommanderCounter.xcodeproj/project.pbxproj

```diff
@@ -378,7 +378,10 @@
 					"-DFOLLY_CFG_NO_COROUTINES=1",
 					"-DFOLLY_HAVE_CLOCK_GETTIME=1",
 				);
-				OTHER_LDFLAGS = "$(inherited)  ";
+				OTHER_LDFLAGS = (
+					"$(inherited)",
+					" ",
```

Comment:
think this is not impactful. usually xcode stuff doens't need to be commited if you're not touching ios config stuff